### ISSUE TITLE
test_region fails on powerpc

### ISSUE THREAD

**Issue by benoit-monin** [gitpython-developers/smmap#25]
The following assert found in [smmap/test/test_util.py](https://github.com/gitpython-developers/smmap/blob/v0.9.0/smmap/test/test_util.py#L85) fails on a powerpc64 system running linux and python 2.7.9: 

    assert rhalfofs.includes_ofs(rofs) and not rhalfofs.includes_ofs(0)

This test probably fails on other non-x86 platforms. And I am not sure it is really meaningful since it is highly dependent on the underlying OS and platform.

**Comment by Byron** [comment]
As this issue only appears in the test-suite, I guess it would be safe to just exclude the said line for power-pc architectures as well, similar to how it's done with `win32`.
Unfortunately I don't really know how to identify such a platform, as I am lacking the hardware. Maybe you can cookup a PR or let me know the values of `sys.platform` and `os.name` ?

**Comment by benoit-monin** [comment]
Sorry, I don't have access to real hardware either. I found the problem when updating the package on [opensuse](https://build.opensuse.org/package/show/home:benoit_monin:branches:devel:languages:python/python-smmap) and I patched out the test as a workaround.

This assert does not really test the correctness of smmap but the implementation of mmap on the platform. So I suggest to remove it from the test-suite.
